CHANGELOG.rst

```diff
@@ -2,16 +2,25 @@
 Change history
 ==============
 
-3.x.x (2025-03-31)
+3.0.2 (2025-04-01)
 ------------------
 
 **Project maintenance**
-
-* Confirm support for Postgres 17 and drop (verified) support for Postgres 12
+* upgraded docker image to debian-bookworm [open-api-framework/#125]
```

Comment:
Formatting seems to be off here, could be due to missing newline between the title and the list?

![afbeelding](https://github.com/user-attachments/assets/f865d6b2-3c61-4928-8d77-9fe0bddd6927)